app/src/main/java/com/example/campusguide/Constants.kt

```diff
@@ -21,4 +21,12 @@ object Constants {
     const val SEARCH_LEFT_BOUND = -74.031124
     const val SEARCH_TOP_BOUND = 45.734999
     const val SEARCH_RIGHT_BOUND = -73.417826
+
```

Comment:
Maybe but the constant in the Event Class that I mentioned in Calendar.kt

Reply:
aye 
github hasn't marked this as outdated but i have moved the constants to their respective classes (since they don't need to be accessed by any other ones, for anyone other than us two reading this) o7